samples/cellular/nrf_cloud_mqtt_multi_service/src/shadow_support_coap.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/logging/log_ctrl.h>
+#include <modem/location.h>
+#include <date_time.h>
+#include <stdio.h>
+#include <net/nrf_cloud.h>
+#include <net/nrf_cloud_coap.h>
+
+#include "shadow_support_coap.h"
+
+LOG_MODULE_REGISTER(shadow_support_coap, CONFIG_MULTI_SERVICE_LOG_LEVEL);
+
+#define COAP_SHADOW_MAX_SIZE 512
+
+static int check_shadow(void)
+{
+	int err;
+	char buf[COAP_SHADOW_MAX_SIZE];
+
+	buf[0] = '\0';
+	LOG_INF("Checking for shadow delta...");
+	err = nrf_cloud_coap_shadow_get(buf, sizeof(buf), true);
+	if (err == -EACCES) {
+		LOG_INF("Not connected yet.");
+	} else if (err) {
+		LOG_ERR("Failed to request shadow delta: %d", err);
+	} else {
+		size_t len = strlen(buf);
+
+		LOG_INF("Delta: len:%zd, %s", len, len ? buf : "None");
+		/* Do something with the shadow delta's JSON data, such
```

Comment:
it doesn't look like MSS cares about the shadow deltas, so is this necessary?
if it is necessary, can a comment be added as to why?

in general, what happens if there are a bunch of shadow deltas and the device never acks them?

Reply:
We have requests to provide an example of how a customer could use the shadow delta to send a "command" to a CoAP device when the device sleeps a lot.

It would go here.  The note "Do something..." is a prompt to the reader.

I plan to improve this (future) by showing how they can parse out something that they send using curl, perhaps.  Unfortunately the frontend only allows for ending of the config section, which I'm not sure we want to encourage the customer to modify, since that is for ATv2.  Perhaps we need to designate a new section within the shadow that the front end will also allow edits for.

Reply:
the config section can and should be used by any application.
no need to create a separate section.